src/__Private/LintRunLSPErrorHandler.php

```diff
@@ -0,0 +1,80 @@
+<?hh // strict
```

Comment:
File name should ideally be `LintRunErrorHandlerLSP` to be consistent with the others

Reply:
Or really just use some more consistency in general (e.g. `Lint` vs `Linter`)